repository_service_tuf/helpers/api_client.py

```diff
@@ -90,38 +92,32 @@ def is_logged(settings: LazySettings):
 
 
 def get_headers(settings: LazySettings) -> Dict[str, str]:
-    if settings.get("AUTH") is False:
-        return {}
-
     server = settings.get("SERVER")
     token = settings.get("TOKEN")
+
     if server and token:
-        token_access_check = is_logged(settings)
+        token_access_check = token_state(settings)
         if token_access_check.state is False:
-            raise click.ClickException(
-                f"{str(token_access_check.data)}"
-                "\n\nTry re-login: 'rstuf --auth admin login'"
-            )
+            if settings.AUTH is True:
+                message = "\n\nTry re-login: 'rstuf --auth admin login'"
+            else:
+                message = "\n\nToken is expired"
 
-        expired_admin = token_access_check.data.get("expired")
-        if expired_admin is True:
             raise click.ClickException(
-                "The token has expired. Run 'rstuf --auth admin login'"
+                f"{str(token_access_check.data)}.{message}"
             )
+
         else:
             headers = {"Authorization": f"Bearer {token}"}
-            response = request_server(
-                server, URL.bootstrap.value, Methods.get, headers=headers
-            )
-            if response.status_code != 200:
-                raise click.ClickException(
-                    f"Unexpected error: {response.text}"
-                )
-    else:
+
+    elif settings.get("AUTH") is True:
         raise click.ClickException(
             "Login first. Run 'rstuf --auth admin login'"
         )
 
+    else:
```

Comment:
At this point it means that we have:
- no `--auth` given or
- no `--token` given
- or both
Since the function is `get_headers` it would make more sense to me to return an exception with a message to give either --auth or --token (e.g., "Provide the `--auth` or `--token` option"), because we are trying to actually get the headers.